tools/vsimporter/xib2nib/UIControl.h

```diff
@@ -42,6 +42,22 @@ enum {
    UIControlEventAllEvents           = 0xFFFFFFFF
 };
 
+typedef enum {
+    UIControlContentHorizontalAlignmentCenter = 0,
+    UIControlContentHorizontalAlignmentLeft = 1,
+    UIControlContentHorizontalAlignmentRight = 2,
+    UIControlContentHorizontalAlignmentFill = 3,
+} _UIControlContentHorizontalAlignment;
+typedef unsigned UIControlContentHorizontalAlignment;
```

Comment:
Why do it this way instead of just typedef enum {.....} UIControlContentHorizontalAlignment?